cves/CVE-2018-1336.yml

```diff
@@ -10,15 +10,15 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-119
```

Comment:
Would you be able to explain your reasoning behind choosing CWE-119?  It looks like that more has to do with allowing to read/write outside of an assigned buffer.  The CVE in question appears to be more of an infinite loop.  Is it that the code overruns its buffer, which _causes_ the loop?